.github/workflows/remote-integ-tests-workflow.yml

```diff
@@ -7,8 +7,8 @@ env:
   # avoid warnings like "tput: No value for $TERM and no -T specified"
   TERM: xterm
   OPENSEARCH_DASHBOARDS_VERSION: 'main'
-  OPENSEARCH_VERSION: '3.0.0-alpha1'
-  OPENSEARCH_PLUGIN_VERSION: '3.0.0.0-alpha1'
+  OPENSEARCH_VERSION: '3.1.0'
+  OPENSEARCH_PLUGIN_VERSION: '3.1.0.0'
```

Comment:
should this point to 3.2?

Reply:
will bump it later. Currently, Dashboards core is still using 3.1. And backend has build failure on 3.2.